src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -13127,15 +13138,24 @@ static void Main()
     }
 }
 ";
-            CreateCompilationWithMscorlib40AndSystemCore(text).VerifyDiagnostics(
-                // (10,41): error CS0853: An expression tree may not contain a named argument specification
-                //         Expression<Func<int>> f = () => new List<int> { 1 } [index: 0];
-                Diagnostic(ErrorCode.ERR_ExpressionTreeContainsNamedArgument, "new List<int> { 1 } [index: 0]").WithLocation(10, 41)
-                );
+            var comp = CreateCompilationWithMscorlib40AndSystemCore(text, parseOptions: TestOptions.Regular.WithLanguageVersion(languageVersion));
+            if (languageVersion == LanguageVersion.CSharp13)
+            {
+                comp.VerifyDiagnostics(
+                    // (10,41): error CS0853: An expression tree may not contain a named argument specification
+                    //         Expression<Func<int>> f = () => new List<int> { 1 } [index: 0];
+                    Diagnostic(ErrorCode.ERR_ExpressionTreeContainsNamedArgument, "new List<int> { 1 } [index: 0]").WithLocation(10, 41));
```

Comment:
>); [](http://example.com/codeflow?start=136&length=2)

Formatting change creates an unnecessary noise in the diff. In other similar places too